decision-records/005-repository-organisation-for-v5.md

```diff
@@ -0,0 +1,122 @@
+# Decision Record Title
```

Comment:
We should replace "Decision Record Title" with "Update our repository structure to prepare for v5" and remove line 3.